tracee/tracee.go

```diff
@@ -169,14 +175,14 @@ func New(cfg TraceeConfig) (*Tracee, error) {
 	t := &Tracee{
 		config: cfg,
 	}
-	ContainerMode := (t.config.Mode == ModeContainerAll || t.config.Mode == ModeContainerNew)
+	ContainerMode := t.config.Filter.ContFilter.Enabled && t.config.Filter.ContFilter.Set
```

Comment:
Pidns is not related to container (although containers create a new pid ns, and that what we had until now)

Reply:
pidns means that I want to capture events only from new pidns's right?

Reply:
Yes. It's different from "container" flag, as "container" flag is true for all conatiner processes, including those that existed in pid namespaces that are not new

Reply:
ok. so we do need to add pidns here. why do you think we don't?

Reply:
As I said, I kept pidns only to be able to trace NEW containers, which is not possible with the container flag, otherwise, I would remove it from other places too.
Also, a process unsharing it's pid namespace is not always a container. (actually, a mount namespace fits much better as a description for a container than pid namespace). We can change the way "container" filter is working in the future, but pidns stands for itself

Reply:
Your answer is missing the context of this discussion. we are talking about this particular line right? all I'm saying is that when printing the table, we should show host ns information if tracing pidns

Reply:
The context of this particular line is **ContainerMode**, how am I missing the context?

Reply:
it's just the argument to `newEventPrinter`

Reply:
Well, this argument also controls the printing of UTS_NAME and MNT_NS, which doesn't exactly fit to PIDNS. I don't mind doing this change though